src/views/OrderDetail.vue

```diff
@@ -129,6 +129,7 @@ export default defineComponent({
     },
     markBopisItem (item: any, event: any) {
       if(this.isBopisItem(item)){
+        // Kept Pickup Store check for orders that have been already marked BOPIS. Will remove after sometime.
```

Comment:
Update the comment to `Need to remove the `Pickup Store` check, currently kept it for backward compatibility.

Reply:
Updated comment